jobs/templates/roleNotifierMail.js

```diff
@@ -0,0 +1,8 @@
+export default (roles) => {
+    let messageBody = "";
+    roles.forEach( (role) => messageBody += `--> role "${role.name}" of color #${role.color} is ${role.isActive ? "" : "in"}active\n`);
```

Comment:
Could it be possible to use map?

Reply:
Surely would look nicer, I tried to use reduce and it didnt work, but we cant use map because it returns an array

Reply:
We need to get only deactivated roles in this report